WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -87,6 +103,14 @@
 
     private final List<PostModel> mTrashedPosts = new ArrayList<>();
 
+    private final List<PostModel> mSearchResults = new ArrayList<>();
+    private PostsListAdapter mSearchListAdapter;
+    private String mSearchTerm;
+    private Handler mHandler;
+    private boolean mIsSearching;
```

Comment:
We need to save all these `mIsSearching`, `mCanSearchMore`, `mSearchOffset` in the bundle if we don't want to lose track of them on config changes (rotation etc).